it/uk/gov/hmrc/incorporatedentityidentificationfrontend/api/controllers/JourneyControllerISpec.scala

```diff
@@ -234,6 +235,200 @@ class JourneyControllerISpec extends ComponentSpecHelper with JourneyStub with I
         )
       }
     }
+    "return captured data" when {
+      "the journeyId exists and verificationStatus is BusinessVerificationFail" in {
+        stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+        stubRetrieveIncorporatedEntityInformation(testJourneyId)(
+          status = OK,
+          body = Json.toJsObject(
+            IncorporatedEntityInformation(
+              companyProfile = testCompanyProfile,
+              optCtutr = Some(testCtutr),
+              identifiersMatch = false,
```

Comment:
Identifiers match should be "true". BusinessVerification is only invoked if the identifiers match.

Reply:
.... it is true .... but there is not check in place ... I did it on purpose ;-) .... the api returns whatever  is stored into IncorporatedEntityInformation without checking data consistency.
I will change it because it is friday ;-)